packages/template-retail-react-app/app/utils/utils.js

```diff
@@ -322,3 +322,11 @@ export const keysToCamel = (obj) => {
 
     return obj
 }
+
+export const mergeArrays = (arr1 = [], arr2 = []) => {
```

Comment:
`mergeArrays` makes me think `[...arr1, ...arr2]`. This is more `mergeMatchedItems` or something. 

Also, this includes items from `arr1` that are not in `arr2`, but does _not_ include items in `arr2` that are not in `arr1`. Is that desired?

Reply:
For the use case that we have, `arr2` is always a subset of `arr1`. I renamed the func and added jsdoc! Good feedback!